TROUBLESHOOTING.md

```
@@ -0,0 +1,1297 @@
+```bash
+# Add to ~/.bashrc or ~/.zshrc
+expo📦 Node.js Version Issues
+
+**Error Messages:**
+```
+The engine "node" is incompatible with this module
+error:0308010C:digital envelope routines::unsupported
+Node Sass does not yet support your current environment
+``
+source ~/.bashrc
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)